starter_code/elevator.js

```diff
@@ -1,19 +1,86 @@
+
 class Elevator {
   constructor(){
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.direction = "UP";
+    this.passengers = [];
+    this.waitingList = [];
+
+ //   let arr = [...this.passengers, ...this.waitingList];
+
   }
 
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
+  start() { 
+    var upd = setInterval(this.update.bind(this),2000);
+  }
+  stop() { 
+    clearInterval(upd);
```

Comment:
La variable upd está fuera del scope de la función, por lo no debería funcionar al llamarla.

Una solución óptima pasaría por crear un nuevo valor en la clase tipo this.updater dentro de start.

Si quieres luego te explico como funcionan los scopes respecto a la aplicación.